examples/sid_visualization/docker-compose.yml

```diff
@@ -25,7 +25,7 @@ x-with-gpus: &with_gpus
 
 services:
   triton:
-    image: nvcr.io/nvidia/tritonserver:22.08-py3
+    image: nvcr.io/nvidia/tritonserver:23.06-py3
```

Comment:
Remember to revert this back since updates to the triton version are being handled by a different PR now.